src/wp-includes/class-wp-theme-json-data.php

```diff
@@ -39,7 +39,7 @@ class WP_Theme_JSON_Data {
 	 * @param array  $data   Array following the theme.json specification.
 	 * @param string $origin The origin of the data: default, theme, user.
 	 */
-	public function __construct( $data = array(), $origin = 'theme' ) {
+	public function __construct( $data = array( 'version' => WP_Theme_JSON::LATEST_SCHEMA ), $origin = 'theme' ) {
```

Comment:
How does it change the `@since`? It sounds like this is fine to do in a follow-up?

Reply:
The change in default value here is more to avoid confusion than anything. The same was done in the constructor for `WP_Theme_JSON`, and the constructor for `WP_Theme_JSON` automatically runs the migration to the latest version.

The `version` option is required for theme.json data and an omission of it originally signaled that the file was the shape of an experimental-theme.json or theme.json v0 https://github.com/WordPress/gutenberg/pull/36154. When it landed in core, it represented a v1 theme.json, so this probably should have been updated then for clarity https://github.com/WordPress/gutenberg/pull/36155/files#diff-b03597cc3da199e5aa5a94950e8241135904853e7c3b82d758e42744878afae7R315-R335.

It didn't really matter that much at the time because the migrations weren't changing default values. Since the default value is changed now, keeping this as an empty array means migrating from a v1 theme.json and adding `'defaultFontSizes' => false` and `'defaultSpacingSizes' => false` which I don't think is the intention of the default value.